build.gradle

```diff
@@ -94,6 +96,43 @@ configure(subprojects.findAll {it.name != 'data-prepper-api'}) {
     }
 }
 
+configure(mavenArtifactProjects) {
+    java {
+        // TODO: Add once Javadocs are working
+        //withJavadocJar()
+        withSourcesJar()
+    }
+
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                from components.java
+                pom {
+                    name = project.name
```

Comment:
What is the project name that gets published here? Are we publishing multiple projects here?

Reply:
It is the name of the current project. Thus, it will `data-prepper-api` and `common`. Though, per the discussion above, I will remove `common`.